MyLibrary.Domain.Integration.Tests/BookRepositoryTest.cs

```diff
@@ -0,0 +1,72 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using MyLibrary.Domain.Entities;
+
+namespace MyLibrary.Domain.Integration.Tests
+{
+    [TestClass]
+    public class BookRepositoryTest
+    {
+        private const string CONNECTIONSTRING = "lalala";
+
+        private const string DATABASE_NAME = "";
+        private IBookRepository _bookRepository;
+        [TestInitialize]
+        public void setup()
+        {
+
+        }
+        [TestCleanup]
+        public void Cleanup()
+        {
+
+        }
+
+        [TestMethod]
+        public void GetAll_ReturnsAllBooks()
+        {
+            var result = _bookRepository.GetAll();
+            Assert.AreNotEqual(null, result);
+        }
+        [TestMethod]
+        public void GetOne_ReturnsBookByTitle()
+        {
+            var result = _bookRepository.GetOne("Title");
+            Assert.IsNotNull(result);
+        }
+        [TestMethod]
+        public void Insert_AddsBook()
+        {
+            var book = new Book
+            {
+                Title = "Test Title",
```

Comment:
So there is no id field on the Book Object, this can cause problems later on, when defining an entity its always best to give it an Id and assume that all could be different, i know there is ISBN there which could be used as an identifier however if you wanted to extend this system to have many versions of the same book (and maybe have a property for condition like 'second-hand') then the ISBN wont be unique anymore closing the app off to expansion or a messy refactor